desktop/src/main/java/bisq/desktop/main/overlays/windows/DisputeSummaryWindow.java

```diff
@@ -132,6 +132,7 @@
     private ChangeListener<Toggle> tradeAmountToggleGroupListener;
     private ChangeListener<String> compensationOrPenaltyListener;
     private boolean updatingUi = false;
+    private boolean payoutPromptOnDisplay = false;
```

Comment:
I see alternative approach in code base in [`DaoStateMonitorView`](https://github.com/bisq-network/bisq/blob/8e3f95383e1c1784cf7cb735b345e2a73b8456c9/desktop/src/main/java/bisq/desktop/main/dao/monitor/daostate/DaoStateMonitorView.java#L65) and [`TradeStepView`](https://github.com/bisq-network/bisq/blob/ee374c3c39720e540a79311d48a1d6daa93d8abd/desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/steps/TradeStepView.java#L106) where `Popup` instance is stored instead of boolean. To be more consistent we could choose single way.

Reply:
Good suggestion!  I'll adopt that approach.